pkg/mdformatter/linktransformer/validator.go

```diff
@@ -32,8 +32,31 @@ func (v GitHubPullsIssuesValidator) IsValid(k futureKey, r *validator) (bool, er
 	return false, nil
 }
 
-// RoundTripValidator.IsValid returns true if url is checked by colly.
+// RoundTripValidator.IsValid returns true if url is checked by colly or it is a valid local link.
 func (v RoundTripValidator) IsValid(k futureKey, r *validator) (bool, error) {
+	matches := remoteLinkPrefixRe.FindAllStringIndex(k.dest, 1)
+	if matches == nil && r.validateConfig.ExplicitLocalValidators {
```

Comment:
Does it make sense to pack it in separate function? Looks like 1:1 to repetition.